ci/qa/phpcs.xml

```diff
@@ -17,4 +16,8 @@
     <file>dev/</file>
     <file>tests/</file>
 
+    <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/TypeHints/DeclareStrictTypesSniff.php" />
```

Comment:
Just a question... when to use the full path and when not for a rule ref?

Reply:
I think I needed a full path to the sniff because it is an external ruleset. Not sure though..